worlds/AutoWorld.py

```diff
@@ -326,7 +325,7 @@ class World(metaclass=AutoWorldRegister):
     settings: ClassVar[Optional["Group"]]
```

Comment:
```suggestion
    settings: "ClassVar[Group | None]"
```
If any of an annotation is in quotes, it should all be in quotes. (But I don't know whether the quotes are even needed, because we have `from __future__ import annotations` at the top.)

Reply:
This still seems to break the settings system as far as I can tell

Reply:
ok - We should fix the code that processes this. But that seems out of scope for this PR.

Reply:
I think it's because of the quotes. I just know I tried it and it failed tests, I didn't look into it.

Reply:
that's what i meant by "that format" :p

Reply:
update: it absolutely doesn't and this looks mean to fix :(
(if you're trying to not make too many assumption *and* take into account both Optional and | formats,)
https://github.com/ArchipelagoMW/Archipelago/blob/main/settings.py#L765-L769